Dt 372 migrate utilities pages layout to typography

juliodialpad opened the pull request:
## Description

We are migrating all the utilities pages, this is the last part

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [ ] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [ ] Ensure all `gulp` scripts successfully compile.
 - [x] Update, remove, or extend all affected documentation.
 - [x] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
 - [x] Request a review from Brad Paugh, David Becher, or Drew Chandler.

## Obligatory GIF (super important!)
![](https://media.giphy.com/media/xT1R9LUBYOXB4b8E6Y/giphy.gif)

braddialpad reviewed (approved):
Looks good, just a question about the build process.

braddialpad commented:
Something seems really messed up with the package-lock. I deleted and reinstalled. Can you pull that down and do an npm install and make sure everything still works for you locally?

juliodialpad commented on docs/.vuepress/clientAppEnhance.js:
```diff
@@ -21,9 +21,12 @@ import IconWarning from "@svgIcons/IconWarning.vue";
+import IconPhone from "@svgIcons/IconPhone.vue";
+
+// Import compiled dialtone to include responsive variations
+import '../../lib/dist/css/dialtone.min.css';
```
Yes, due to the .min.css, I'll change it to use the none minified version, but we are going to change the gulp script anyway, this is just a temporary solution.

braddialpad replied:
alright, fine for now, we can merge this and finish off configuration changes in the future. We can talk about this at grooming tomorrow.

juliodialpad commented:
> Something seems really messed up with the package-lock. It kept freezing on build unless I deleted and rebuilt it. pushed this new one to this branch. Can you pull that down and do an npm install and make sure everything still works for you locally?

It's messed up, when I updated my package.json with that one, I got a warning

`warning an anonymous plugin or theme was detected and ignored`

And it doesn't detects the theme at all, I'm checking it to fix it.

braddialpad commented:
> > Something seems really messed up with the package-lock. It kept freezing on build unless I deleted and rebuilt it. pushed this new one to this branch. Can you pull that down and do an npm install and make sure everything still works for you locally?
> 
> It's messed up, when I updated my package.json with that one, I got a warning
> 
> `warning an anonymous plugin or theme was detected and ignored`
> 
> And it doesn't detects the theme at all, I'm checking it to fix it.

hmm wonder if there was a breaking change in a newer version of vuepress

juliodialpad commented:
> > > Something seems really messed up with the package-lock. It kept freezing on build unless I deleted and rebuilt it. pushed this new one to this branch. Can you pull that down and do an npm install and make sure everything still works for you locally?
> > 
> > 
> > It's messed up, when I updated my package.json with that one, I got a warning
> > `warning an anonymous plugin or theme was detected and ignored`
> > And it doesn't detects the theme at all, I'm checking it to fix it.
> 
> hmm wonder if there was a breaking change in a newer version of vuepress

I think so, the `config.ts` has some errors now

ninamarina reviewed (approved):
LGTM

Previously:
<img width="427" alt="image" src="https://github.com/user-attachments/assets/18e1dccf-c317-468e-ae0a-d2644421af3b">

Now:
<img width="290" alt="image" src="https://github.com/user-attachments/assets/2654653b-a8b9-4a6c-91ac-ed646ca68e96">

francisrupert reviewed (approved):
Yes, much better.
